frontend/src/components/custom-tools/tools-main/ToolsMain.jsx

```diff
@@ -162,7 +162,12 @@ function ToolsMain() {
     <div className="tools-main-layout">
       <div className="doc-manager-header">
         <div className="tools-main-tabs">
-          <Tabs activeKey={activeKey} items={items} onChange={onChange} />
+          <Tabs
+            activeKey={activeKey}
+            items={items}
+            onChange={onChange}
+            moreIcon={<></>}
```

Comment:
@jagadeeswaran-zipstack Any reason why we're passing empty fragment here?

Reply:
moreIcon takes only ReactNode, and the issue that Shuveb noted that causes CPU utilisation can be resolved with this. This happens from time to time in the Antd Tab component.